nextcloudappstore/core/models.py

```diff
@@ -106,6 +106,7 @@ class Meta:
         verbose_name = _('App Release')
         verbose_name_plural = _('App Releases')
         unique_together = (('app', 'version'),)
+        ordering = ['-version']
```

Comment:
Does this work with semantic releases? AFAIK it uses natural ordering so 13.0.0 is considered lower than 2.0.0